project/Dependencies.scala

```diff
@@ -42,11 +45,26 @@ object Dependencies {
 
   val coreModuleDeps = Seq(
     scalaJSDOM,
-    chandu0101Macros
+    chandu0101Macros,
+    npmDependencies in Compile += "react" -> Version.react
   )
 
-  val webModuleDeps = Seq(
+  val universalModuleDeps = Seq(
+    npmDependencies in Compile ++= Seq(
+      "react" -> "15.4.0-rc.4",
+      "react-native" -> "0.38.0"
```

Comment:
in that case its ok. In past scalajs-jquery facade shipped with jsDeps(jQuery), soon users of library faced issues ..  , thats why i asked :)